anti-analysis/packer/gopacker/gopacker_capa.yml

```diff
@@ -0,0 +1,29 @@
+rule:
+  meta:
+    name: packed with GoPacker
+    namespace: anti-analysis/packer/gopacker
+    author: jared.wilson@fireeye.com
+    description: The sample appears to be packed with GoPacker.
+    scope: file
+    att&ck:
+    - Defense Evasion::Obfuscated Files or Information [T1027.002]
+    mbc:
+      - Anti-Static Analysis::Software Packing
```

Comment:
this should have an ID with it, i think

Reply:
It's not, it's actually the default extension given to packed files. I'll add a Description.

Reply:
```suggestion
      - Anti-Static Analysis::Software Packing [F0001]
```